webpack.config-test.js

```diff
@@ -9,7 +9,7 @@ module.exports = {
       {
         enforce: "pre",
         test: /\.js$/,
-        exclude: /node_modules/,
+        exclude: /node_modules|\.test\.js$/,
```

Comment:
What's the motivation for disabling lints on tests?

Reply:
Answering both questions here.  The reason was that the quality of the test files wasn't very good and a ton of warnings and errors were produced.  This is no longer the case and the ignore directives have been lifted.  Thanks for pointing out this oversight!

Reply:
Spoke too soon. The test file is still producing a ton of errors and warnings. Will attempt to include test files in linting checks in the next PR since it removes jQuery as a dependency and the test infrastructure is overhauled.

Reply:
Further to the above, I can confirm that tests are now linted in the [upcoming PR](https://github.com/dossier/html-highlighter/tree/remove-jquery); see [https://github.com/dossier/html-highlighter/commit/14387d7f0e3167596a5efa1d8a2348dfdf23c510](latest commit).

Reply:
Great! Thanks.